karma.conf.js

```diff
@@ -16,9 +16,10 @@ module.exports = function(config) {
 
         // list of files / patterns to load in the browser
         files: [
-            "node_modules/mocha/mocha.js",
             "node_modules/chai/chai.js",
+            "node_modules/esprima/dist/esprima.js",
```

Comment:
Once #11398 is merged, these files from "node_modules" can be removed from `karma.conf.js`.